.github/workflows/push.yaml

```
@@ -0,0 +1,84 @@
+name: OpenFisca-Core / Pull request review
+
+on:
+  pull_request:
+    types: [assigned, opened, reopened, synchronize, ready_for_review]
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-22.04, windows-2019]
+        numpy: [2.1.1, 1.24.2]
+        python: [3.11.9, 3.9.13] # Patch version must be specified to avoid any cache confusion, since the cache key depends on the full Python version. If left unspecified, different patch versions could be allocated between jobs,  and any such difference would lead to a cache not found error.
+        include:
+        - os: ubuntu-22.04
+          activate_command: source venv/bin/activate
+        - os: windows-2019
+          activate_command: .\venv\Scripts\activate
+    uses: ./.github/workflows/_before.yaml
+    with:
+        os: ${{ matrix.os }}
+        numpy: ${{ matrix.numpy }}
+        python: ${{ matrix.python }}
+        activate_command: ${{ matrix.activate_command }}
+
+  test:
+    needs: [setup]
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-22.04, windows-2019]
+        numpy: [2.1.1, 1.24.2]
+        python: [3.11.9, 3.9.13] # Patch version must be specified to avoid any cache confusion, since the cache key depends on the full Python version. If left unspecified, different patch versions could be allocated between jobs,  and any such difference would lead to a cache not found error.
```

```suggestion
        python: [3.11.9, 3.9.13]
```
I suggest to keep the command only on the first matrix of this file, for readability.